aws/utility.sh

```diff
@@ -715,16 +715,23 @@ function create_snapshot() {
 
 # -- Git Repo Management -- 
 function clone_git_repo() {
-  local repo_url="$1"; shift
+  local repo_provider="$1"; shift
+  local repo_host="$1"; shift
+  local repo_path="$1"; shift
   local repo_branch="$1"; shift
   local local_dir="$1"; 
 
-  [[ (-z "${repo_url}") ||
+  [[  (-z "${repo_provider}") ||
+      (-z "${repo_host}") ||
+      (-z "${repo_path}") ||
       (-z "${repo_branch}") ||
       (-z "${local_dir}") ]] && fatalMandatory && return 1
 
   trace "Cloning the ${repo_url} repo and checking out the ${repo_branch} branch ..."
```

Comment:
is repo_url still defined?

Reply:
Its built in the function so that I can include the credentials, the trace was in the wrong spot.